yt_dlp/extractor/taptap.py

```diff
@@ -0,0 +1,209 @@
+                'uploader': '乌酱',
+                'thumbnail': r're:^https?://.*\.(png|jpg)',
+            }
+        }]
+    }, {
+        'url': 'https://www.taptap.cn/moment/521630629209573493',
+        'info_dict': {
+            'id': '521630629209573493',
+            'title': '《崩坏：星穹铁道》黄泉角色PV——「你的颜色」',
+            'description': 'md5:2c81245da864428c904d53ae4ad2182b',
+            'timestamp': 1711425600,
+            'upload_date': '20240326',
+            'uploader': '崩坏：星穹铁道',
+        },
+        'playlist_count': 1,
+        'playlist': [{
+            'info_dict': {
+                'id': '4006511',
+                'ext': 'mp4',
+                'title': '《崩坏：星穹铁道》黄泉角色PV——「你的颜色」',
+                'description': 'md5:2c81245da864428c904d53ae4ad2182b',
+                'duration': 173,
+                'timestamp': 1711425600,
+                'upload_date': '20240326',
+                'uploader': '崩坏：星穹铁道',
+                'thumbnail': r're:^https?://.*\.(png|jpg)',
+            }
+        }]
+    }]
+
+
+class TapTapAppIE(TapTapBaseIE):
+    _VALID_URL = r'https?://www\.taptap\.cn/app/(?P<id>\d+)'
+    _INFO_API = 'https://www.taptap.cn/webapiv2/app/v4/detail'
+    _ID_PATH = (('app_videos', 'videos'), ..., 'video_id')
+    _META_PATH = {
+        'title': ('title', {str}),
+        'description': ('description', 'text', {str}, {clean_html}),
+    }
+    _TESTS = [{
+        'url': 'https://www.taptap.cn/app/168332',
+        'info_dict': {
+            'id': '168332',
+            'title': '原神',
+            'description': 'md5:e345f39a5fea5de2a46923f70d5f76ab',
+        },
+        'playlist_count': 2,
+        'playlist': [{
+            'info_dict': {
+                'id': '4058443',
+                'ext': 'mp4',
+                'title': '原神',
+                'description': 'md5:e345f39a5fea5de2a46923f70d5f76ab',
+                'duration': 26,
+                'thumbnail': r're:^https?://.*\.(png|jpg)',
+            }
+        }, {
+            'info_dict': {
+                'id': '4058462',
+                'ext': 'mp4',
+                'title': '原神',
+                'description': 'md5:e345f39a5fea5de2a46923f70d5f76ab',
+                'duration': 295,
+                'thumbnail': r're:^https?://.*\.(png|jpg)',
+            }
+        }]
+    }]
+
+
+class TapTapAppIntlIE(TapTapAppIE, TapTapIntlBase):
+    _VALID_URL = r'https?://www\.taptap\.io/app/(?P<id>\d+)'
+    _INFO_API = 'https://www.taptap.io/webapiv2/i/app/v5/detail'
+    _DATA_PATH = ('data', 'app')
+
+
+class TapTapPostIntlIE(TapTapAppIntlIE):
+    _VALID_URL = r'https?://www\.taptap\.io/post/(?P<id>\d+)'
+    _INFO_API = 'https://www.taptap.io/webapiv2/creation/post/v1/detail'
+    _INFO_QUERY_KEY = 'id_str'
+    _DATA_PATH = ('data', 'post')
+    _ID_PATH = ((('videos', ...), 'pin_video'), 'video_id')
+    _META_PATH = {
+        'timestamp': ('published_time', {int_or_none}),
+        'uploader': ('user', 'name', {str}),
```

Comment:
Again the additional metadata fields
```suggestion
        'modified_timestamp': ('edited_time', {int_or_none}),
        'uploader': ('user', 'name', {str}),
        'uploader_id': ('user', 'id', {int}, {str_or_none}),
```